config/config.txt

```diff
@@ -0,0 +1,46 @@
+[default]
```

Comment:
this is an example file of simply config, and should not be here, if necessary, add relevant information to the config in the example folder of simply